src/vtvalidate.cpp

```diff
@@ -76,7 +93,8 @@ struct AsyncHelloWorker : Nan::AsyncWorker {
         // The try/catch is critical here: if code was added that could throw an
         // unhandled error INSIDE the threadpool, it would be disasterous
         try {
-            result_ = do_expensive_work(louder_);
+            // What if tile is gzipped? Does std::string still make sense for the buffer param?
+            result_ = parseTile(tile_);
```

Comment:
Now passing the [buffer data_view reference](https://github.com/mapbox/vtvalidate/blob/f0a99d721865015fb4be532c72f6c586df45ab37/src/vtvalidate.cpp#L111) instead of a vector_tile object.